packages/react-next/src/components/SwatchColorPicker/SwatchColorPicker.types.ts

```diff
@@ -45,24 +46,30 @@ export interface ISwatchColorPickerProps {
    */
   colorCells: IColorCellProps[];
 
+  /**
```

Comment:
Can't directly comment on it, but the docs for `selectedId` need to be updated like this:
```
  /**
   * ID of the current selected color swatch. Only provide this if the SwatchColorPicker is a
   * controlled component where you are maintaining its current state; otherwise, use the
   * `defaultSelectedId` property.
   */
```